app/controllers/admin/events/list.js

```diff
@@ -132,7 +132,15 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       this.notify.success(this.l10n.t('Event has been deleted successfully.'));
     } catch (e) {
       console.warn(e);
-      this.notify.error(this.l10n.t('An unexpected error has occurred.'));
+      if (e.constructor === Array && 'errors' in e) {
```

Comment:
Hmm, For an array it'll be true, but `'errors' in e` will be false

For `{ errors: [] }`, vice versa, so it'll always resolve to false?